packages/atlas/src/components/_comments/CommentRow/CommentRow.stories.tsx

```diff
@@ -2,14 +2,11 @@ import { Meta, Story } from '@storybook/react'
 import React from 'react'
 import { BrowserRouter } from 'react-router-dom'
 
-import { Text } from '@/components/Text'
-import { cVar } from '@/styles'
-
-import { CommentRow, CommentRowProps } from './CommentRow'
+import { Comment, CommentProps } from '../Comment'
 
 export default {
-  title: 'comments/CommentRow',
-  component: CommentRow,
+  title: 'comments/Comment',
+  component: Comment,
```

Comment:
but this is a CommentRow, not a Comment. We have stories for comment already. This is a different thing.